# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/libwebrtcglue/WebrtcMediaDataEncoderCodec.h
# Commit: d838dc7d2bb8
# Full Hash: d838dc7d2bb89ab5c9c2e8e93895db1c9310560f
# Author: Dan Minor <dminor@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 4
# Description:
#   Bug 1654112 - Update include paths for moved upstream code. r=ng
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D130059
# ==============================================================================

diff -r 587c40771588 -r d838dc7d2bb8 dom/media/webrtc/libwebrtcglue/WebrtcMediaDataEncoderCodec.h
--- a/dom/media/webrtc/libwebrtcglue/WebrtcMediaDataEncoderCodec.h	Fri Oct 23 13:49:51 2020 -0400
+++ b/dom/media/webrtc/libwebrtcglue/WebrtcMediaDataEncoderCodec.h	Fri Oct 23 14:02:54 2020 -0400
@@ -11,7 +11,7 @@
 #include "PlatformEncoderModule.h"
 #include "WebrtcGmpVideoCodec.h"
 #include "common_video/include/bitrate_adjuster.h"
-#include "webrtc/modules/video_coding/include/video_codec_interface.h"
+#include "modules/video_coding/include/video_codec_interface.h"
 
 namespace mozilla {
 